cwatm/hydrological_modules/water_quality/erosed.py

```diff
@@ -39,15 +40,107 @@ class waterquality_erosed(object):
     def __init__(self, model):
         self.var = model.var
         self.model = model
+        self.waterquality_vars = waterquality_vars(model)
 
     def initial(self):
         """
                 INITIAL PART OF THE EROSED MODULE
+
+        Sediment yield per grid cell is calculated with the Modified Universal Soil Loss Equation (MUSLE)
+        Williams (1995)
         """
-        print('erosed init')
+        self.waterquality_vars.initial()
+        # load initial MUSLE maps
+        # exponential function for fraction of rock (exp-0.053 * fcr)
+        self.var.fcfr = 1
+
+        # K_usle: USLE soil erodibility factor
+        self.var.kFactor = self.var.Kf = loadmap('kFactor')
```

Comment:
Line 58: self.var.Kf is already taken  in the Phosphorus module. Please use another variable name.

Reply:
I do not know how this variable name got in there.  It must have been an accidental autocomplete. 
The variable name is now self.var.kFactor. Please let me know if I should find a completely different name.

Reply:
Have you removed the self.var.Kf, if it is not necessary?

Reply:
I have removed it with the latest commit/pull request.